documentation/learningPath/aks.md

```diff
@@ -2,3 +2,58 @@
 ### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2FlearningPath%2Faks)
 
 # AKS
+
+In addition to its availability as a .NET CLI tool, the `dotnet monitor` tool is available as a prebuilt Docker image that can be run in container runtimes and orchestrators, such as Kubernetes. To learn more about using `dotnet monitor` with Kubernetes, you can check out our [example deployment](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/kubernetes.md) and our [video tutorial](https://github.com/dotnet/dotnet-monitor/tree/main/samples/AKS_Tutorial). This section covers how to test your development version of `dotnet monitor` in AKS; however, we recommend having some basic experience with AKS before following this workflow.
+
+## What This Workflow Does
+
+This workflow takes your local development copy of `dotnet-monitor`, patches it with a local development copy of the [.NET Core Diagnostics Repo](https://github.com/dotnet/diagnostics#net-core-diagnostics-repo), and makes it available as an image for you to consume in an ACR (Azure Container Registry). Note that there are many other ways to do this - this is meant to serve as a basic template that can be adapted to match your needs.
+
+1. Open `pwsh` and run the [generate-dev-sln script](https://github.com/dotnet/dotnet-monitor/blob/main/generate-dev-sln.ps1), providing a path to your local copy of the diagnostics repo.
+
+>**Note**: If your changes do not involve the [.NET Core Diagnostics Repo](https://github.com/dotnet/diagnostics#net-core-diagnostics-repo), you don't need to complete this step.
+
+```bash
+cd C:\your-path\dotnet-monitor 
+.\generate-dev-sln.ps1 C:\your-path\diagnostics
+```
+
+2. Publish `dotnet monitor` to your desired (local) target location (TEMP is used throughout this example)
+
+```bash
+dotnet publish .\src\Tools\dotnet-monitor -o $env:TEMP\dotnet-monitor -c Release -f net6.0
+```
+
+3. Add a Dockerfile to the `dotnet monitor` directory created in the previous step. Below is an example of the contents of the Dockerfile:
+
+```bash
```

Comment:
This gets .NET Monitor running in a container but it's not nearly as representative of what the actual .NET Monitor image is. It's missing many environment variables (behavior is going to be different), the entrypoint is different and lacks command (so those who are overriding it in their orchestrators are going to have mismatches), the profiler is missing, etc.

So either all of these caveats should be documented (so folks don't come us asking why it's not working as they expected), or the workflow should be updated to produce something more representative of the real image. For the latter, this would include:
1. Updating the publishing instructions to do a `.\build.cmd -ci -archive`.
1. Effectively copying https://github.com/dotnet/dotnet-docker/blob/main/src/monitor/7.0/alpine/amd64/Dockerfile and making minor adjustments for copying the built archive instead of downloading .NET Monitor.

This doesn't solve the missing profiler since that requires a Linux build (so folks doing this on Linux are in good shaped).

Reply:
After thinking about this more and discussing it, I think we should use your publish instructions because that will get most of .NET Monitor scenarios working without a lot of hassle. However, I think using a Dockerfile that is more representative of what is actually shipped would be beneficial in eliminating functional differences between the local image vs the official image.